pandas/core/tools/numeric.py

```diff
@@ -158,12 +158,13 @@ def to_numeric(arg, errors='raise', downcast=None):
                     if values.dtype == dtype:
                         break
 
+    from pandas import Series, Index
```

Comment:
Let's just import `Series` and `Index` when we need to (i.e. import `Series` under `is_series` and `Index` under `is_index`.